modules/tinymce/src/themes/silver/main/ts/ui/context/ContextUi.ts

```diff
@@ -51,7 +51,7 @@ const renderContextToolbar = (spec: { onEscape: () => Optional<boolean>; sink: A
 
     inlineBehaviours: Behaviour.derive([
       AddEventsBehaviour.config('context-toolbar-events', [
-        AlloyEvents.runOnSource<EventArgs>(NativeEvents.transitionend(), (comp, _se) => {
+        AlloyEvents.runOnSource<EventArgs<TransitionEvent>>(NativeEvents.transitionend(), (comp, se) => {
```

Comment:
No real idea, but since we pulled the transitions code out, should this still be here?